### DISCUSSION TITLE
2024-06-21(금)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#201] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
프로필 페이지 토스트 적용

### 해야할 일
프로필 이미지 삽입 PR
my order 페이지 API 연동

### 어려운 일
프로필 이미지 삽입 이슈... 웨안뒈...

**Comment by devwqc** [comment]
### 완료한 일
- 자잘한 수정들

### 해야할 일
- `SortButtonModal`
- 홈, 추천상품, 인기상품, 검색, 찜 페이지

### 어려운 일
- 파이팅팅팅!!💪

**Comment by q45402sk** [comment]
### 완료한 일
purchaseDeliveryPage

### 해야할 일
상품상세 api
deliveryAddPage
deliveryEdiitPage api

### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
optionBottomSheet 상품 옵션 조합 

### 어려운 일
optionBottomSheet
